src/background.ts

```diff
@@ -1 +1,14 @@
+import { browser } from "webextension-polyfill-ts";
+import 'webext-dynamic-content-scripts';
+import addDomainPermissionToggle from "~utils/domain-accept";
+
+addDomainPermissionToggle();
+
+browser.runtime.onInstalled.addListener((details) => {
+    if (details.reason === "install") {
+        browser.tabs.create({ url: "https://www.gitpod.io/#extension-activation?track=true" });
+    }
+});
+browser.runtime.setUninstallURL("https://www.gitpod.io/#extension-uninstall?track=true");
+
```

Comment:
This we accidentally removed in v2 and so are bringing it back in this PR. Before merging, the `#` just have to be removed here – didn't want to track my uninstalls during testing.

Reply:
Didn't we use to have a proper landing page for install and uninstall?

Reply:
Ah, got it. the `#` confused me. 👍

Reply:
That would be an excellent place to point to the docs as well for configuring self-hosted SCMs